### DISCUSSION TITLE
程序运行后，一直是转动界面

### DISCUSSION THREAD

**Discussion by veancn** [reorproject/reor#488] [Q&A]
程序安装后运行时，一直是转动界面，未有进度条指示。删除程序重新安装，但不让重装进行软件设置，还是直接进入启动界面，还是不停的转动。如何能彻底卸载、重新安装并能重新设置后，再来运行这个程序。谢谢！ 
![微信图片_20241114173109](https://github.com/user-attachments/assets/da2c6d1c-c840-4726-abc6-17ee41a2ce18)
![Uploading 截图1.png…]()

**Comment by starsang1995** [comment]
有可能是因为连不上网，你有没有试试看打开VPN的全局模式然后重启一下程序？
